opencga-app/app/scripts/azure/arm/solr/azuredeploy.json

```diff
@@ -330,7 +331,17 @@
             "managedDisk": {
               "storageAccountType": "[parameters('diskType')]"
             }
-          }
+          },
+          "dataDisks": [
+            {
+                "diskSizeGB": 1023,
```

Comment:
Do we need to think about sizing here? Guessing 1TB is going to be a reasonable size for the smaller deployments but would this need to vary with the TShirt sizes?

Probably not one to pick up in this PR

Reply:
Agree, and note this is under 1TB, so won't get the 1TB performance.